cpp/src/io/avro/avro_common.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
I'm not sure about changing copyright year just because of changing file name?

Reply:
The copyright enforcement script will require this.

Reply:
the script passed, looks like it's not required for renames?